azerothcore/azerothcore-wotlk
# fix(Scripts): Pilfering Perfume

## Changes Proposed:
-  Swapped the credit spell for a quest complete credit core side to prevent losing the aura

## Issues Addressed:
- Closes #4550



## SOURCE:



## Tests Performed:

- Compiled and tested on windows 10

## How to Test the Changes:

Horde:
.event start 8
.quest add 24536
.quest complete 24536
.go c id 37172
Get the next quest and check if it works.

## Known Issues and TODO List:

None

## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
